### ISSUE TITLE
Release version without upper bound on FSharp.Core

### ISSUE THREAD

**Issue by TysonMN** [destructurama/fsharp#23]
The constraint on `FSharp.Core` was relaxed in PR #13.  However, a (non pre-) release has not been created that includes that change.  The latest release version available on nuget.org is [`1.1.0`](https://www.nuget.org/packages/Destructurama.FSharp/1.1.0), which still requires `FSharp.Core` to have a version strictly less than `4.7.0`.  The two prereleases on nuget.org after version `1.1.0` are versions [`1.1.1-dev-00033`](https://www.nuget.org/packages/Destructurama.FSharp/1.1.1-dev-00033) and [`1.1.1-dev-00035`](https://www.nuget.org/packages/Destructurama.FSharp/1.1.1-dev-00035), and neither includes restricts `FSharp.Core` from above.

Can a new release of `Destructurama.FSharp` be made that does not include the upper bound constraint on `FSharp.Core`?

**Comment by ScottShingler** [comment]
Ran into this issue today. We have a .NET Core 3.1 project that requires `FSharp.Core 4.7.2` but are stuck on `Destructurama.FSharp 1.0.14` because of the version constraint in `Destructurama.FSharp 1.1.0`. As a result, we get the following warning:

```text
warning NU1701: Package 'Destructurama.FSharp 1.0.14' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETCoreApp,Version=v3.1'. This package may not be fully compatible with your project.
```

We are averse to relying on dev / preview versions, so it would be great to have a stable version that relaxes the `FSharp.Core` version constraint.

**Comment by AlexeyRaga** [comment]
Any chance of having a new release with relaxed upper bounds?

**Comment by nblumhardt** [comment]
I've drafted #27 for this; needs a final check-over :-)

**Comment by AlexeyRaga** [comment]
@nblumhardt Thanks, can't wait! :)

**Comment by sungam3r** [comment]
https://www.nuget.org/packages/Destructurama.FSharp/1.2.0 was released on 30.09.2021
